setup.py

```diff
@@ -107,8 +107,11 @@
         "-DPROTOTYPES=1",
         "-DHAVE_STRCHR",
         "-DHAVE_MEMCPY",
-        "-Dieee754_float32_t=float",
         "-DHAVE_MPGLIB",
+        # Some data types are defined in config.h, which we don't include:
+        "-Dieee754_float32_t=float",
+        "-Duint32_t=u_int32_t",
+        "-Duint16_t=u_int16_t",
```

Comment:
This feels a little dangerous. [cstdint](https://en.cppreference.com/w/cpp/types/integer) already defines a lot of these so wouldnt you risk overwriting those defines? What config.h are you referring to?